python/.changelog.d/1745.incompatible.2

```diff
@@ -0,0 +1 @@
+`trezorlib.stellar` now relies on stellar-sdk, please check the README.md for more information.
```

Comment:
why is this an incompatible change entry?

Reply:
Because the lib did not rely on it before, if the user directly upgrades trezorlib without installing stellar-sdk, then he cannot use trezorlib.stellar normally. Do you have any other suggestions?

Reply:
I see. OK, let's leave this.